Reference: https://github.com/beetbox/mediafile/issues/109
test/test_mediafile.py::OpusTest::test_read_audio_properties fails with mutagen 1.48.1
---
diff --git a/test/test_mediafile.py b/test/test_mediafile.py
index a6be425..95158e4 100644
--- a/test/test_mediafile.py
+++ b/test/test_mediafile.py
@@ -987,7 +987,7 @@ class OpusTest(ReadWriteTestBase, unittest.TestCase):
     extension = "opus"
     audio_properties = {
         "length": 1.0,
-        "bitrate": 66792,
+        "bitrate": 52392,
         "bitrate_mode": "",
         "encoder_info": "",
         "encoder_settings": "",
